src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -336,6 +336,8 @@ public override void Dispose()
             this.logger.LogInformation("Disposing finalized block info repository.");
             this.finalizedBlockInfoRepository.Dispose();
 
+            this.logger.LogInformation("Disposing address indexer.");
```

Comment:
Delete this log

Reply:
I'll need it while syncing mainchain and then shutting the node down to see if it takes too long to save addresses

Reply:
ok just remember to remove it later